view/frontend/templates/js/replacejs.phtml

```diff
@@ -352,7 +352,6 @@ if (!$block->isOnPageFromWhiteList() && !$block->isMinicartEnabled()) { return;
             var publishableKey = getCheckoutKey();
             if (scriptTag) {
                 scriptTag.setAttribute('data-publishable-key', publishableKey);
-                createOrder();
```

Comment:
The whole `insertConnectScript` function seems redundant. The connect.js is already included on every page.

Reply:
I agree with you, but let me handle this in another PR.